public/js/dom.js

```diff
@@ -1413,6 +1413,149 @@ export function loadPython() {
+    simulationWrapperDiv.appendChild(simulationTitleDiv)
+
+    const simulationIdDiv = document.createElement('div')
+    simulationIdDiv.setAttribute('class', 'simulation-id')
+    simulationIdDiv.innerText = `Unique simulation ID: ${simulationId}`
+    simulationWrapperDiv.appendChild(simulationIdDiv)
+
+    simulationWrapperDiv.appendChild(simulationInputWrapperDiv)
+    simulationWrapperDiv.appendChild(simulationOutputWrapperDiv)
+
+    // Display input parameters in the input simulation wrapper div
+    displayInputParameters(
+        simulationInputWrapperDiv,
+        inputParameters,
+        simulationId,
+        MODES.simple.name
+    )
+
+    // Display reports in the output simulation wrapper div
+    const reportsTitleDiv = document.createElement('h3')
+    reportsTitleDiv.innerText = 'Summary reports (output)'
+    simulationOutputWrapperDiv.appendChild(reportsTitleDiv)
+
+    summaryReports.forEach((report, index) => {
+        displayReportUnified(
+            simulationOutputWrapperDiv,
+            report,
+            simulationId,
+            index,
+            report.dimensionsString
+        )
+    })
+
+    simulationWrapperDiv.scrollIntoView({ block: 'end' })
+}
+
+
+
+function displayReportUnified(
+    parentDomEl,
+    report,
+    simulationId,
+    simulationNo,
+    keyCombinationDisplay
+) {
+    const { noiseMetrics, data, measurementGoal, scalingFactor } = report
+    const { noise_ape_percent, noise_rmsre } = noiseMetrics
+
+    // Display report table title
+    const titleDiv = document.createElement('h4')
+    titleDiv.innerText = 'Measurement goal: ' + measurementGoal
+    parentDomEl.appendChild(titleDiv)
+    // Display noise
+    displayNoiseAsPercentage(parentDomEl, noise_ape_percent, noise_rmsre)
+    // Display details section title
+    displayDataDetailsTitle(parentDomEl)
+    parentDomEl.appendChild(document.createElement('br'))
+    // Display dimensions
+    displayDimensionsInOutput(parentDomEl, keyCombinationDisplay)
+    // Display scaling factor
+    displayScalingFactor(parentDomEl, scalingFactor)
+
+    // Display table containing report data
+    const dataTableTitle = document.createElement('h6')
+    dataTableTitle.innerText = 'Data table:'
+    parentDomEl.appendChild(dataTableTitle)
+    const tableId = `output-data-table-${simulationId}-${measurementGoal}-${simulationNo}`
+    const detailsDiv = document.createElement('details')
+    detailsDiv.setAttribute('id', tableId)
+    detailsDiv.setAttribute('class', 'offset-left')
+    parentDomEl.appendChild(detailsDiv)
+
+    // Generate data table
+    const table = new TabulatorFull(`#${tableId}`, {
+        data,
+        // Create columns from data field names
+        autoColumns: true,
+        layout: 'fitColumns',
+        pagination: true,
+        paginationSize: 5,
+    })
+
+    // Save table temporarily; used for XLSX multi-table download
+    tempSaveTable_simpleMode(table, `${simulationId}-${measurementGoal}`)
```

Comment:
Doesn't this break the download function for the advanced mode?
If so: let's consider replacing the current implementation of the temp tables with a cross-modes unified object for all tables. That object should be emptied (reset to `{}`) upon mode switch.
WDYT?

Reply:
This seems ok for the advanced mode. 
However it seems the downloadAll is broken for advanced only

Reply:
Ok, fixed this with the latest commit - we have different "tempSaveTable_<mode>" because the table id s have different structure. We could just use the same structure for that as well.
For now I fixed by checking current mode and using the corresponding structure.